CONTRIBUTING.md

```diff
@@ -143,35 +143,139 @@ The CI is testing that the changes are inline with our standard styling.
 GitHub has a helpful page on
 [getting started with writing and formatting on GitHub](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github).
 
+### Style guide
+
+There certain style rules we are trying to follow in the way the specifications are written.
+
+Many of those styling issues can fixed automatically using a linter: see 
+the section[Fixing Remark errors from Travis](#fixing-travis-remark-errors).
+
+Some others need to fixed manually:
+
+- Do not use latin abbreviations like `"e.g"`, `"i.e"`, `"etc"` that can be extra confusing
+  to some readers and try to replace them by common english equivalent 
+  (`"for example"`, `"that is"`, `"and so on"`).
+
+#### Soft rules
+
+We are also trying to follow certain "soft rules" in the way we format the specifications in markdown.
+
+Those rules are sometimes for internal consistency in terms of styling and aesthetics,
+but they several of them are also there because they help the workflow of 
+tracking changes, reviewing them on Github and making suggestions.
```

Comment:
```suggestion
Those rules are sometimes for internal consistency in terms of styling and aesthetics,
but several of them are also there because they help the workflow of 
tracking changes, reviewing them on GitHub and, making code suggestions.
```